docs/codeql/codeql-for-visual-studio-code/customizing-settings.rst

```diff
@@ -46,13 +46,17 @@ The query history **Format** setting controls how the extension lists queries in
 
 To override the default label, you can specify a different format for the query history items.
 
+.. _configuring-settings-for-running-queries:
```

Comment:
What does this do? Does this add an anchor to the heading? (Sorry for the stupid question)

Reply:
Yep! It's an anchor, so we can refer to it with the `:ref:` tag below 😃